application/controllers/admin/globalsettings.php

```diff
@@ -62,6 +62,44 @@ public function refreshAssets()
         }
     }
 
+    public function sendTestEmail()
+    {
+        //Use the current user details for the default administrator name and email for this survey
+        $user = User::model()->findByPk(Yii::app()->session['loginID']);
+        $owner = $user->attributes;
+
+        $sTo = $owner['full_name'] ." <".$owner['email'].">";
+        $sFrom = Yii::app()->getConfig("siteadminname")." <".Yii::app()->getConfig("siteadminemail").">";
+        $sSubject = sprintf(gT('Test email from '), Yii::app()->getConfig('sitename'));
+        $sSiteAdminBounce = Yii::app()->getConfig('siteadminbounce');
+
+        $body   = array();
+        $body[] = sprintf(gT('This is a test email from '), Yii::app()->getConfig('sitename'));
```

Comment:
This needs to be properly fixed to use %s. Check out the documentation on sprintf.

Reply:
Resolved in latest commit